eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/beaconv2/BeaconServiceV2.java

```diff
@@ -146,7 +147,7 @@ public BeaconAlleleResponse find(String chromosome, Long start, Long startMin, L
 
     private String checkParameters(String chromosome, String referenceBases, Long start, Long end,
                                    String alternateBases, String variantType, String assemblyId,
-                                   String includeDatasetResponses) {
+                                   String includeDatasetResponses) throws IllegalArgumentException {
```

Comment:
The `throws` is now declared, but the internal behavior of the method is the same. What I meant was for instance:

```java
if (assemblyId == null || assemblyId.length() == 0) {
            throw new IllegalArgumentException("Please provide an assembly identifier");
}
```

If no exceptions are thrown, then it means the method completed successfully.

Reply:
Okay, I will remove the `throws` :)

Reply:
Fixed :)